spec/presenters/psul_index_presenter_spec.rb

```diff
@@ -22,15 +22,19 @@
 
       let(:config) do
         Blacklight::Configuration.new.configure do |config|
-          config.add_index_field 'publication_display_ssm'
+          config.add_index_field 'author_person_display_ssm'
+          config.add_index_field 'author_corp_display_ssm'
+          config.add_index_field 'author_meeting_display_ssm'
           config.add_index_field 'format'
+          config.add_index_field 'publication_display_ssm'
+          config.add_index_field 'edition_display_ssm'
           config.add_index_field 'url_fulltext_display_ssm'
         end
       end
 
-      it 'has no label for publication statement and format but still has a label for URL' do
-        expect(presenter.fields['publication_display_ssm'].label).to be_nil
+      it 'has no label for format and publication statement but still has a label for URL' do
         expect(presenter.fields['format'].label).to be_nil
+        expect(presenter.fields['publication_display_ssm'].label).to be_nil
```

Comment:
Hmm, is there a reason to test `format` and `publication_display_ssm` to `be_nil` but not `author_person_display_ssm`, `author_corp_display_ssm`, `author_meeting_display_ssm`, and `edition_display_ssm`. I guess it seems like either test for one field or test for them all? 🤷‍♂️

Reply:
There were only those fields on search results when I first wrote that spec so no specific reason. I can add the others  👍

Reply:
Oh I remember now why not testing for others because the document does not actually have those fields.

Reply:
Ok, so I guess why didn't you mock these other fields too? Like, to me it makes sense to just mock one example of a field that shouldn't have a label and test that _or_ mock and test them all. Does that make sense?